deploy/olm-catalog/ibm-iam-operator/3.9.1/ibm-iam-operator.v3.9.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1991 @@
+      - rules:
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - mutatingwebhookconfigurations
+          #iam.hooks.securityenforcement.admission.cloud.ibm.com used by oidcclient-watcher 
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+          # Investigate further whether we can avoid the creation of clusteroles and clusterrolbeindings
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+          - escalate
+          - bind  
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - user.openshift.io
+          resources:
+          - users
+          # The default admin user has to be created
+          verbs:
+          - create
+          - get
+          - list
+          - patch
+          - update            
+          - watch
+          - delete
+        serviceAccountName: ibm-iam-operator
+      deployments:
+      - name: ibm-iam-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: ibm-iam-operator
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                productID: 068a62892a1e4db39641342e592daa25
+                productMetric: FREE
+                productName: IBM Cloud Platform Common Services
+              labels:
+                app.kubernetes.io/instance: ibm-iam-operator
+                app.kubernetes.io/managed-by: ibm-iam-operator
+                app.kubernetes.io/name: ibm-iam-operator
+                name: ibm-iam-operator
+            spec:
+              affinity:
+                nodeAffinity:
+                  requiredDuringSchedulingIgnoredDuringExecution:
+                    nodeSelectorTerms:
+                    - matchExpressions:
+                      - key: kubernetes.io/arch
+                        operator: In
+                        values:
+                        - amd64
+                        - ppc64le
+                        - s390x
+              containers:
+              - command:
+                - ibm-iam-operator
+                env:
+                - name: ICP_PLATFORM_AUTH_IMAGE
+                  value: quay.io/opencloudio/icp-platform-auth:3.4.0
+                - name: ICP_IDENTITY_PROVIDER_IMAGE
+                  value: quay.io/opencloudio/icp-identity-provider:3.4.0
+                - name: ICP_IDENTITY_MANAGER_IMAGE
```

Comment:
@rashmi43 you need to change the image version here as well for identity provider and manager

Reply:
done